docs/api.md

```diff
@@ -51,7 +51,7 @@ let counterTransform = createTransform(
   { whitelist: 'reducerA'}
 )
 
-persistStore(store, { transforms: counterTransform })
+persistStore(store, { transforms: [ counterTransform ] })
```

Comment:
Can you remove the spaces around `counterTransform` here? That'd make it consistent with the rest of the docs :)